public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -268,16 +268,15 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel)
     controls: [
       new DashboardControls({
         variableControls: [new VariableValueSelectors({}), ...filtersSets, new SceneDataLayerControls()],
-        timeControls: Boolean(oldModel.timepicker.hidden)
-          ? []
-          : [
-              new SceneTimePicker({}),
-              new SceneRefreshPicker({
-                refresh: oldModel.refresh,
-                intervals: oldModel.timepicker.refresh_intervals,
-              }),
-            ],
+        timeControls: [
```

Comment:
They always exist but are not rendered, so the state is maintained. This is the main change of this PR

Reply:
Yup yup, this is good. This is a second instance when this proves to be a better way. It simplifies edit a lot to have even empty objects ready in the hierarchy.